lib/methods/import.js

```diff
@@ -294,6 +340,8 @@ module.exports = self => {
             continue;
           }
 
+          // TODO: We should ask user too if he want to override singleton docs
+          // This logic might be useless since we pre check duplicates
```

Comment:
It does work now if the ID is the same and will ask if the user wants to override, since came because we now pre check duplicates for all documents, singletons included.
There is a ticket to make it work even if the ID isn't the same so I don't go further about this here.